site/sfguides/src/improvado/improvado.md

```diff
@@ -0,0 +1,183 @@
+
+![snowsight_databases](assets/snowsight_databases.png)
+
+
+3. Select the necessary schema to where you’ve setup Data Load to see a list of its tables. 
+
+![snowsight_tables](assets/snowsight_tables.png)
+
+
+4. To query your data, you’ll need to create a new SQL Worksheet. First, go to the Projects > Worksheets page. Click the Plus button and select SQL Worksheet from the dropdown. 
+
+![snowsight_worksheets](assets/snowsight_worksheets.png)
+
+5. Here you can select the necessary table that you want to query using SQL and see its structure.
+
+![snowsight_sql1](assets/snowsight_sql1.png)
+
+6. Check the data loaded to this table in the Results view or create a Chart to help you visualize it.
+
+![snowsight_sql2](assets/snowsight_sql2.png)
+
+
+
+
+<!-- ------------------------ -->
+## Data Privacy
+Duration: 2
+
+In this step, we will learn about Data Privacy policies and rules within Improvado & Snowflake.
+
+Improvado AI Agent is compatible with any Row Access Policies you’ve set up in Snowflake. Improvado enforces data governance policies such as access controls and quality standards right from the metadata level. This ensures that your data governance workflows are consistent and reliable.
+
+You can check Improvado’s [Privacy Policy](https://improvado.io/company-legal/privacy-policy) for more details.
+
+### User access management in Improvado
+We also provide tools to manage user access within the Improvado platform.
+
+Workspaces allow for access management and control within a single Improvado instance. This is especially helpful if you want to separate data by specific Products or Client accounts, ensuring a structured and simplified experience.
+
+![Workspaces](assets/workspaces.png)
+
+
+<!-- ------------------------ -->
+## Data Access and Consumption
+Duration: 2
+
+The AI Agent automatically scans and collects metadata for each table, column and row values in your Snowflake storage. This step is crucial as it enables the AI Agent to understand the structure and relationships within your database.
+
+Improvado’s integration with Snowflake ensures that your analytics, machine learning, and ETL services are all connected within a unified platform. This integration simplifies the data management process and enhances the overall efficiency of your data workflows.
+
+By leveraging these capabilities, Improvado facilitates efficient data access and consumption, providing you with the tools necessary to extract valuable insights and maintain robust data governance practices.
+
+### Billing dashboard
+Improvado Billing Dashboard offers an in-depth analysis of your data usage, providing crucial segmented insights into extracted rows over your usage period. It also includes interactive features, enhancing your ability to examine and understand the data in a more dynamic and effective manner.
+
+![Billing dashboard](assets/billing_dashboard.png)
+
+
+<!-- ------------------------ -->
+## Conclusion

```

Comment:
Rename this to "Conclusion And Resources"

Reply:
Fixed